index.js

```diff
@@ -45,8 +93,30 @@ htmlTarget2.innerHTML = answer2
 //where: name, saves the name of the player. Lives, represents the remaining oportunities each time the player fails. Fail_numbers, is an array of numbers that stores the fail numbers the player has used
 
 //@return {string} win / gameOver => the string that says if the user wasted the three oportunities showing the fails numbers or the name if the player wins
+
+let player = {
+  player_name: prompt('Type your name.'),
+  lives: 3, 
+  fail_numbers: []
+};
+let guess;
+const randomNumber = Math.round(Math.random() * 40 + 10);
+
 guessTheNumber  = () => {
```

Comment:
currently, this function does not `return` any value but it should return a string stating if the user won or lost